server/src/main/java/org/elasticsearch/index/mapper/DocumentMapper.java

```diff
@@ -18,9 +18,18 @@
 import org.elasticsearch.index.IndexVersion;
 import org.elasticsearch.index.IndexVersions;
 
+import java.time.Instant;
 import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
 
 public class DocumentMapper {
+    static final NodeFeature INDEX_SORTING_ON_NESTED = new NodeFeature("mapper.index_sorting_on_nested");
+    static final long ERROR_THROTTLING_INTERVAL_SECONDS = 60;
+
+    private static final AtomicBoolean lastErrorLogLock = new AtomicBoolean(false);
```

Comment:
nit: let's not call it a "lock" since it isn't

Reply:
But it is - in the sense that an integer can be a counter.